### PR TITLE
Fix Ingest due ASF_Search Updates

### LINKED ISSUE #79: Invalid datetime values in delivery json metadata in v0.2.0
GUNW products generated using DockerizedTopsApp v0.2.0 have invalid `sensing_start` and `sensing_stop` values in their `.json` metadata files. This causes these products to be rejected when attempting to ingest them into the ASF archive.

For example, from https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/819086fb-3fb1-4294-8cb2-c1effb453f52/S1-GUNW-D-R-087-tops-20221204_20221122-161627-00157W_00020N-PP-37a3-v2_0_6.json :

```
"sensing_start": "2022-12-04T16:16:12.000ZZ",
"sensing_stop": "2022-12-04T16:16:42.000ZZ",
```

This is due to a change in [asf_search v5.0.0](https://github.com/asfadmin/Discovery-asf_search/blob/master/CHANGELOG.md#500):
> Timestamps in geojson fields now include an explicit Z time zone indicator.

The new build of DockerizedTopsApp pulled in the new version of asf_search, but also still appends an additional `Z` to the timestamps  at https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/main/isce2_topsapp/delivery_prep.py#L134

### PR DESCRIPTION
**Pull request by cmarshak**
Resolves #79 for production.

The latest merge into dev modifies the output json and is validated correctly as can be seen below.

```
import json
import jsonschema
import requests

resp_json = requests.get('https://hyp3-a19-jpl-contentbucket-1wfnatpznlg8b.s3.us-west-2.amazonaws.com/a6bdf7a4-7dc6-472c-bebf-3b1c10ac82ff/S1-GUNW-D-R-044-tops-20210925_20200930-172127-00169W_00053N-PP-812d-v2_0_6.json').json()

with open('schema.json') as f:
    schema = json.load(f)

jsonschema.validate(instance=resp_json, schema=schema)
```

### CODE REVIEW

**Comment by cmarshak** [comment]
I am never sure if we have to modify the changelog at this point because of two merges into dev or we can keep it as such.

I am happy if I just manually modify release to `0.2.1`. Let me know.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 
+## [0.2.2]
+
+* Fixes write of start/stop sensing times due to changes in ASF Search v5.0.0 (see #79); schema requires 6 trailing digits
+
+
+## [0.2.1]
+
+* Fixes write of start/stop sensing times due to changes in ASF Search v5.0.0 (see #79)
+
```
Comment (jhkennedy) [inline CHANGELOG.md:17]: Yes, these should be combined. If there's more than one release past whatever is shown here:
https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/releases/latest

The second PR should have just added the changes to the unreleased version. This is a common mistake, unfortunately, so I might have to figure out how to adjust the changelog actions to make it easier.

**Comment by cmarshak** [comment]
I just changed it directly - honestly if this is the hardest thing - I can get the hang of it!

Still actions are super efficient IMHO.

**Review by jhkennedy — APPROVED** [review]
looks good to me!

**Pull request merged**